Dockerfile

```diff
@@ -95,7 +95,7 @@ ENTRYPOINT ["/usr/bin/tini", "--", "/usr/local/bin/mvn-entrypoint.sh"]
 CMD ["/openmrs/startup-dev.sh"]
 
 ### Production Stage
-FROM tomcat:8.5-$RUNTIME_JDK
```

Comment:
Did you make the change in this file accidentally?

Reply:
This change occurred automatically/accidentally. I will revert it

Reply:
@dkayiwa  This change was done in a different PR

Reply:
So let us remove it from this pull request.